src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -1,115 +1,172 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* Copyright (c) 2021-2022 FIRST. All Rights Reserved. */
```

Comment:
In general, we probably shouldn't change copyright notices unless they are ours.

Reply:
Fixed it, but changed the line endings slightly (so the formatter doesn’t mess with them)